Ms2Database/DBClasses/Ms2DBContext.cs

```diff
@@ -11,8 +11,10 @@ public class Ms2DbContext : DbContext
     {
         public Ms2DbContext() : base("name=DatabaseConnectionString") // Creates context for database via connection string
         {
-            //Database.SetInitializer<Ms2DBContext>(new DropCreateDatabaseAlways<Ms2DBContext>());
+            //Database.SetInitializer<Ms2DbContext>(new DropCreateDatabaseAlways<Ms2DbContext>());
```

Comment:
Is this just temporarily commented out? May want to add a TODO comment just in case.

Reply:
This is temporarily commented out. It basically recreates the database on each run. Which I use consistently to test database structuring. Once The whole database is finished structuring. This line of code will be gone.